### ISSUE TITLE
[Bug]: 0x5f3's config for genre Musical returns wrong list

### ISSUE THREAD

**Issue by sthyregod** [meisnate12/Plex-Meta-Manager-Configs#564]
### Link to File

https://github.com/meisnate12/Plex-Meta-Manager-Configs/blob/master/0x5f3/metadata/movie/5_genre_popular.yml

### Version Number

1.19.0

### What branch are you on?

master

### Describe the Bug

The "Popular in ..." collections returns the wrong list for musicals. IMDb identifies the list as singular as shown:

```
https://www.imdb.com/search/title/?title_type=feature,tv_movie&genres=musical&languages=en&sort=moviemeter,asc
```

But the config creates the URL as plural:

```
https://www.imdb.com/search/title/?title_type=feature,tv_movie&genres=musicals&languages=en&sort=moviemeter,asc
```
Notice that the returned list from this URL shows unfiltered popular movies while the correct URL only shows musicals

Fixing it should be as simple as changing `genres: musicals` to `genres: musical` on line 158.

It looks like "Top rated in ..." suffers the same issue.

**Comment by cpt-kuesel** [comment]
@0x5f3 can you take a look at this?

**Comment by cpt-kuesel** [comment]
Merged the fix in #566
